packages/react-data-grid-addons/src/cells/headerCells/filters/AutoCompleteFilter.js

```diff
@@ -66,21 +66,22 @@ export default class AutoCompleteFilter extends React.Component {
 
   handleChange(value) {
     const filters = value;
-    this.setState({ filters });
     this.props.onChange({ filterTerm: filters, column: this.props.column, rawValue: value, filterValues: this.filterValues });
   }
 
   render() {
+    const { value, multiSelection } = this.props;
+
     return (
       <Select
         autosize={false}
         name={`filter-${this.props.column.key}`}
         options={this.state.options}
-        placeholder={this.state.placeholder}
+        placeholder="Search"
```

Comment:
To API standpoint, maybe we should do `placeholder = {this.props.placeHolder != null ? this.props.placeholder : 'Search'}`

Reply:
I think we should look into deprecating the `addons` package because of the following reasons. 
- It is not very well maintained and does not address all the requirements from the community
- Many of the addons are dependent on third party libraries like `react-select` or `react-dnd` and it is extra work to keep them upto date.
- We should continue improving the API so it is easy to customize it but do not provide any custom implementation. I think eventually we should move all the editors/filters etc to the demos folder but do not publish them. 

wdyt?

Reply:
I'm in favor of removing the `addons` packages as it doesn't provide any value other than examples in my perspective. It's not powerful enough for day to day use that I barely use it except the dragging which we cannot avoid.